src/components/activities-test.js

```diff
@@ -240,6 +242,40 @@ describes.realWin('Activity Components', {}, (env) => {
         expect(original).to.be.instanceof(WebActivityPorts);
       });
     });
+
+    describe('openIframe', () => {
+      it('adds sut and publicationId', async () => {
+        const callMock = sandbox
+          .mock(activityPorts, 'openActivityIframePort_')
+          .expects('openActivityIframePort_')
+          .withExactArgs(
+            iframe,
+            `${url}?sut=${TOKEN}&publicationId=${publicationId}`,
+            {}
+          )
+          .once();
+        await activityPorts.openIframe(iframe, url, {});
+        callMock.verify();
+      });
+
+      it('does not duplicate already existing sut and publicationId', async () => {
```

Comment:
I'd split this into 2 separate tests, 1 for each `if` statement (existing sut, does not replace; existing pulicationId, does not replace). If the logic below is updated to 
```
if (swgUserToken && !queryParams.has('sut') && pubId && !queryParams.has('publicationId') ) { ... }
```
both these tests pass. Also, we should have a test making sure that if the swgUserToken is empty from storage, it should not add to the url